neuvue_project/workspace/plugins.py

```diff
@@ -1,7 +1,11 @@
-import abc
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from typing import Dict, Any
+import requests
+from django.conf import settings
+from nglui.statebuilder import StateBuilder, AnnotationLayerConfig, PointMapper, site_utils
```

Comment:
can you update the requirements.txt with the version of nglui in your environment, getting an error importing `site_utils`